src/execution/__tests__/executor-test.ts

```diff
@@ -330,6 +333,7 @@ describe('Execute: Handles basic execution tasks', () => {
     const document = parse('query Example { a }');
     const rootValue = { contextThing: 'thing' };
 
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
Please create an issue with links to such places so we don't forget to fix it also add link to issue in `eslint-disable` comments.